app/tests/core/reducer_spec.js

```diff
@@ -0,0 +1,72 @@
+import {List, Map} from 'immutable';
+import {expect} from 'chai';
+import reducer from '../../src/reducer';
+
+describe('reducer', () => {
```

Comment:
I put all reducer tests into one describe, as I don't think nesting them makes sense here, as we only have on test per action so far. Or should I change it?